src/test/java/com/nulabinc/zxcvbn/JavaPortTest.java

```diff
@@ -0,0 +1,93 @@
+package com.nulabinc.zxcvbn;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import javax.script.Invocable;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Map;
+
+@RunWith(Parameterized.class)
+public class JavaPortTest {
+    private static ScriptEngine engine;
+    private String password;
+    private final Zxcvbn zxcvbn;
+
+    public JavaPortTest(String password) {
+        this.password = password;
+        zxcvbn = new Zxcvbn();
+    }
+
+    @BeforeClass
+    public static void initEngine() {
+        ScriptEngineManager manager = new ScriptEngineManager();
+        engine = manager.getEngineByName("nashorn");
```

Comment:
@sm4
The Java environment set in .travis.yml is java 1.7. Using Nashorn requires java 1.8. Try passing string arg "js" to ScriptEngineManager#getEngineByName.

`
engine = manager.getEngineByName("js");
`